release/ray_release/cluster_manager/minimal.py

```diff
@@ -20,7 +20,7 @@ class MinimalClusterManager(ClusterManager):
     Builds app config and compute template but does not start or stop session.
     """
 
-    @retry((ClusterEnvCreateError), delay=10, jitter=5, tries=2)
+    @retry.retry(init_delay_sec=10, jitter_sec=5, max_retry_count=2)
```

Comment:
can we add the supported list of Exceptions here too, so that it only retries on `ClusterEnvCreateError`? .. I actually don't know how `retry` did it, but catching all `Exception` is an anti-pattern. like it should not retry on typo caused `TypeError`

really, for this single use case, it might be easier to just add a small while loop here ..

Reply:
Sure, added.